mentoring/public/css/dashboard.css

```diff
@@ -0,0 +1,31 @@
+.pb-dashboard table {
```

Comment:
@sarina `pb-dashboard` is the block name :-)

pb- is a prefix for all the blocks in this repository, and dashboard is the name of this block.

`pb` is short for "Problem Builder", which is the proposed new name for the mentoring XBlock.